### PR TITLE
ONC-4623: First Example of tool

### PR DESCRIPTION
**Pull request by codecodecoder78**

### CODE REVIEW

**Comment by codecodecoder78** [comment]
@ruslan-oncoshot let me know if there are any changes to the structure/documentation needed

File: src/upload.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                    console.log(err)
+                }
+            })
+
+
+
+        })
+    })
+}
+console.log("This tool uploads the formatted JSON file to the staff API.\n")
```
Comment (ruslan-oncoshot) [inline src/upload.js]: Oncoshot API

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "url": "git+https://github.com/Oncoshot/Profiles-Import-Tool.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Oncoshot/Profiles-Import-Tool/issues"
+  },
+  "homepage": "https://github.com/Oncoshot/Profiles-Import-Tool#readme",
+  "dependencies": {
+    "csv-parser": "^3.0.0",
```
Comment (ruslan-oncoshot) [inline package.json]: no need

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "env-cmd": "^10.1.0",
+    "prompt": "^1.1.0",
```
Comment (ruslan-oncoshot) [inline package.json]: let's not use prompt

File: package.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "profiles-import-tool",
+  "version": "1.0.0",
+  "description": "This tool is an example how to import profiles to Oncoshot via its API",
+  "main": "index.js",
```
Comment (ruslan-oncoshot) [inline package.json]: ?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,26 @@
 # Profiles-Import-Tool
 This tool is an example how to import profiles to Oncoshot via its API
+
+This tool is written in node.js. The API documentation is available on https://api.oncodevel.com/swagger/index.html
```
Comment (ruslan-oncoshot) [inline README.md]: oncoshot not oncodevel

**Review by ruslan-oncoshot — COMMENTED** [review]
instead of prompt can we use command line arguments? it allows piping scripts together

**Review by ruslan-oncoshot — APPROVED** [review]

**Pull request merged**